gtsam/geometry/triangulation.h

```diff
@@ -499,6 +501,8 @@ TriangulationResult triangulateSafe(const CameraSet<CAMERA>& cameras,
 // Vector of Cameras - used by the Python/MATLAB wrapper
 using CameraSetCal3Bundler = CameraSet<PinholeCamera<Cal3Bundler>>;
 using CameraSetCal3_S2 = CameraSet<PinholeCamera<Cal3_S2>>;
+using CameraSetCal3Fisheye = CameraSet<PinholeCamera<Cal3Fisheye>>;
+using CameraSetCal3Unified = CameraSet<PinholeCamera<Cal3Unified>>;
```

Comment:
Similar question. Maybe we should pare down and only list here what's absolutely required in triangulation. Caller can include.